[2025-03-11T15:31:05.640+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ingest_bronze_layer_dag.prepare_dag_execution manual__2025-03-11T15:31:01+00:00 [queued]>
[2025-03-11T15:31:05.647+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ingest_bronze_layer_dag.prepare_dag_execution manual__2025-03-11T15:31:01+00:00 [queued]>
[2025-03-11T15:31:05.647+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-03-11T15:31:05.655+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): prepare_dag_execution> on 2025-03-11 15:31:01+00:00
[2025-03-11T15:31:05.658+0000] {standard_task_runner.py:57} INFO - Started process 18818 to run task
[2025-03-11T15:31:05.661+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ingest_bronze_layer_dag', 'prepare_dag_execution', 'manual__2025-03-11T15:31:01+00:00', '--job-id', '501', '--raw', '--subdir', 'DAGS_FOLDER/ingesting_pipeline.py', '--cfg-path', '/tmp/tmpxyoq2tmm']
[2025-03-11T15:31:05.663+0000] {standard_task_runner.py:85} INFO - Job 501: Subtask prepare_dag_execution
[2025-03-11T15:31:05.694+0000] {task_command.py:416} INFO - Running <TaskInstance: ingest_bronze_layer_dag.prepare_dag_execution manual__2025-03-11T15:31:01+00:00 [running]> on host 41ccd9d1769c
[2025-03-11T15:31:05.908+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Andrew Tran' AIRFLOW_CTX_DAG_ID='ingest_bronze_layer_dag' AIRFLOW_CTX_TASK_ID='prepare_dag_execution' AIRFLOW_CTX_EXECUTION_DATE='2025-03-11T15:31:01+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-11T15:31:01+00:00'
[2025-03-11T15:31:05.917+0000] {base.py:73} INFO - Using connection ID 'postgres_warehouse_db' for task execution.
[2025-03-11T15:31:05.921+0000] {warnings.py:109} WARNING - /opt/***/dags/ingesting_pipeline.py:136: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)

[2025-03-11T15:31:05.929+0000] {python.py:194} INFO - Done. Returned value was: {'next_task': 'extract_and_load_purchases', 'transform_task': 'transform_silver_full_refresh'}
[2025-03-11T15:31:05.937+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ingest_bronze_layer_dag, task_id=prepare_dag_execution, execution_date=20250311T153101, start_date=20250311T153105, end_date=20250311T153105
[2025-03-11T15:31:05.970+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-11T15:31:06.066+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
